plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue

```diff
@@ -461,22 +489,37 @@ export default defineComponent({
         user: this.user,
       });
     },
-    inviteUser(password: string) {
+    async inviteUser(password: string) {
       this.isSavingUserInfo = true;
-      return AjaxHelper.post(
-        {
-          method: 'UsersManager.inviteUser',
-        },
-        {
-          userLogin: this.theUser.login,
-          email: this.theUser.email,
-          initialIdSite: this.firstSiteAccess ? this.firstSiteAccess.id : undefined,
-          passwordConfirmation: password,
-        },
-      ).catch((e) => {
+
+      try {
+        await Promise.all(
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          this.componentExtensionRef.map((component: any) => {
```

Comment:
Would `Component | ComponentReference` work here instead of any as the type for the callback param?